unattended-upgrade

```diff
@@ -220,8 +220,9 @@ class UnattendedUpgradesCache(apt.Cache):
             self.adjust_candidate(self[pkg_name])
 
         if function == apt.package.Package.mark_upgrade \
-            and not pkg.is_upgradable:
-            if not apt_pkg.config.find_b("Unattended-Upgrade::Allow-downgrade", False):
+                and not pkg.is_upgradable:
```

Comment:
Please squash this commit to the previous one to keep tests happy in every commit.

Reply:
Done. Sorry for a long delay...